CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 4.3.0
+- feat: Update Sentry SDKs to 7.35.0 (#631)
+- fix: Replay should be re-exported (#633)
+- test: Test localhost URLs are not modified and stack traces are correctly parsed (#626)
+- CI: A few CI updates and fixes (#625)
+
```

Comment:
```suggestion
## 4.3.0

- feat: Update Sentry SDKs to 7.35.0 (#631)
- fix: Replay should be re-exported (#633)

```

Think we can prob remove the test/ci items, as users shouldn't need to know this.